editions/tw5.com/tiddlers/widgets/ButtonWidget.tid

```diff
@@ -45,6 +45,7 @@ The content of the `<$button>` widget is displayed within the button.
 |dragTiddler |An optional tiddler title making the button draggable and identifying the payload tiddler. See DraggableWidget for details |
 |dragFilter |An optional filter making the button draggable and identifying the list of payload tiddlers. See DraggableWidget for details |
 |tabindex |Sets the `tabindex` attribute of the button to the given value |
+|disabled|Optional, disables the button if set to "yes". Defaults to "no"|
```

Comment:
IMO `<<.from-version "5.1.23">>` is missing

Reply:
good catch, thanks. Added.